echo/serve.go

```diff
@@ -160,3 +165,19 @@ func ModifyKratosRequestHeaders(next echov4.HandlerFunc) echov4.HandlerFunc {
 		return next(c)
 	}
 }
+
+func Properties(c echov4.Context) error {
+	ctx := c.Request().Context().(context.Context)
+
+	props, err := db.GetProperties(ctx)
```

Comment:
Can we make this return a list 

```
properties:
   - name: 
      value: 
      description:
      type: (string|bool|duration|int) 
      source: (local|db)
```

For now the description and type can be empty and string

Reply:
```
[
  {
    "description": "",
    "name": "check.http.disabled",
    "source": "db",
    "type": "",
    "value": "true"
  },
  {
    "description": "",
    "name": "check.kubernetes.disabled",
    "source": "local",
    "type": "",
    "value": "true"
  }
]

```